__tests__/app.test.js

```diff
@@ -331,20 +331,45 @@ describe("PATCH /api/articles/:article_id", () => {
         expect(body.msg).toBe("Bad Request");
       });
   });
-   test("404: Responds with Not Found if the article_id does not exist", () => {
-     return request(app)
-       .patch("/api/articles/2000")
-       .expect(404)
-       .then((res) => {
-         expect(res.body.msg).toEqual("Not Found");
-       });
-   });
-   test("400: Responds with Bad Request when passed an article_id that is not a number", () => {
-     return request(app)
-       .patch("/api/articles/notANumber")
-       .expect(400)
-       .then((res) => {
-         expect(res.body.msg).toBe("Bad Request");
-       });
-   });
+  test("404: Responds with Not Found if the article_id does not exist", () => {
+    return request(app)
+      .patch("/api/articles/2000")
+      .expect(404)
+      .then((res) => {
+        expect(res.body.msg).toEqual("Not Found");
+      });
+  });
+  test("400: Responds with Bad Request when passed an article_id that is not a number", () => {
+    return request(app)
+      .patch("/api/articles/notANumber")
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe("Bad Request");
+      });
+  });
+});
+
+describe.only("DELETE /api/comments/:comment_id", () => {
+  test("200: Deletes the comment of the specified comment_id", () => {
```

Comment:
I really really like this test, you could also just check that the body is an empty object as an additional extra step but I love that you're adding confidence by checking the number of comments has reduced for the article it was associated with. Just check the test description status code.